ext/http/01_http.js

```diff
@@ -471,6 +474,17 @@ function upgradeHttp(req) {
   return req[_deferred].promise;
 }
 
+function upgradeHttp2(req) {
+  return upgradeHttp2Inner(req);
+}
+
+async function upgradeHttp2Inner(req) {
+  const inner = toInnerRequest(req);
+  const res = await core.opAsync("op_http_upgrade_early", inner[streamRid]);
+  // TODO(mmastrac): We're missing the remote address properties here
+  return new Deno.Conn(res, null, null);
```

Comment:
We should address it before landing. I think we can cut corner here - let's pass `conn` parameter to `upgradeHttp2` function and take `remoteAddr` and `localAddr` from it when constructing `Deno.Conn` here.

Reply:
Excellent idea.